tests/performance/load-tests/load-test.sh

```diff
@@ -63,6 +77,17 @@ function parseArguments() {
   done
 }
 
+function cleanup() {
+  echo "Clean up the environment"
+  kubectl delete dw --all >/dev/null 2>&1
```

Comment:
isn't it too risky, shoud we explicitly specify namespace that should be used

Reply:
Done https://github.com/eclipse/che/pull/22901#issuecomment-2027409891